web/yo/app/scripts/controllers/nav.js

```diff
@@ -49,10 +49,10 @@ angular.module('oncokbApp')
         }
         $firebaseAuth().$onAuthStateChanged(function(firebaseUser) {
             if (firebaseUser) {
-                $rootScope.isSignedIn = true;
                 user.setRole(firebaseUser).then(function() {
+                    $rootScope.isAuthorizedUser = true;
```

Comment:
At line 65, you need to handle the failure method.

Reply:
I'll let it send email to dev team.